arbitrum-docs/how-arbitrum-works/timeboost/gentle-introduction.mdx

```diff
@@ -0,0 +1,142 @@
+| `roundDurationSeconds`  | Duration of time that the sequencer will honor the express lane privileges for transactions signed by the current round’s express lane controller.                                                                             | 60 seconds                                 |
+| `auctionClosingSeconds` | Time before the start of the next round. The autonomous auctioneer will not accept bids during this time interval.                                                                                                             | 15 seconds                                 |
+| `beneficiary`           | Address where proceeds from the Timeboost auction are sent to when `flushBeneficiaryBalance()` gets called on the auction contract.                                                                                            | An address controlled by the chain's owner |
+| `_biddingToken`         | Address of the token used to make bids in the Timeboost auction. It can be any ERC20 token (assuming the token address chosen does not have fee-on-transfer, rebasing, transfer hooks, or otherwise non-standard ERC20 logic). | WETH                                       |
+| `nonExpressDelayMsec`   | The artificial delay applied to the arrival timestamp of non-express lane transactions _before_ the non-express lane transactions are sequenced.                                                                               | 0.2 seconds, or 200 milliseconds           |
+| `reservePrice`          | The minimum bid amount accepted by the auction contract for Timeboost auctions, denominated in `_biddingToken`.                                                                                                                | None                                       |
+| `_minReservePrice`      | A value that must be equal to or below the `reservePrice` to act as a "floor minimum" for Timeboost bids. Enforced by the auction contract.                                                                                    | 0.001 `WETH`                               |
+
+## Who is Timeboost for, and how do I use it?
+
+Timeboost is an optional addition to an Arbitrum chain’s infrastructure, meaning that enabling Timeboost is at the discretion of the chain owner and that an Arbitrum chain can fully function normally without Timeboost.
+
+When enabled, Timeboost is meant to serve different groups of parties with varying degrees of impact and benefits. Let’s go through them below:
+
+#### For regular users:
+
+Timeboost will have a minimal impact. Non-express lane transactions will be delayed by a nominal 200ms, which means to the average user, their transactions will take up to 450ms to be sequenced (up from 200ms). Regular users will remain protected from harmful MEV activity, such as sandwich attacks and front-running.
```

Comment:
> their transactions will take up to 450ms to be sequenced
Is it "up to" or is it "at least"? If there's a lot of traffic, I think my transaction might take longer than 250ms to be sequenced (before applying the delay)

Reply:
i think its "approximately" since we cannot guarantee anything. I'll change it
```suggestion
Timeboost will have a minimal impact. Non-express lane transactions will be delayed by a nominal 200ms, which means to the average user, their transactions will take approximately 450ms to be sequenced and included into a block (up from approximately 200ms). All users will remain protected from harmful MEV activity, such as sandwich attacks and front-running, through the continued use of a private mempool.
```